src/main/scala/org/deepdive/extraction/ExtractorRunner.scala

```diff
@@ -400,7 +398,7 @@ class ExtractorRunner(dataStore: JsonExtractionDataStore, dbSettings: DbSettings
 
     // NEW: for mysqlimport compatibility, the file basename must be same as table name.
     val queryOutputFile = new File(queryOutputPath + s"${outputRel}.copy_query_${funcName}.tsv")
-    val gpFileName = s"${outputRel}_unload_${funcName}"
+    val gpFileName = s"${outputRel}_query_unload"
```

Comment:
I made this change because sometimes table names are too long (more than 64 chars) with the previous naming convention, and in this case the table will be incorrectly created with a wrong name, and some procedures in DeepDive will crash. Sorry for not commenting it in the code.